external/docker-integration-tests/pom.xml

```diff
@@ -106,6 +106,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.bundles.repackaged</groupId>
+      <artifactId>jersey-guava</artifactId>
+      <version>2.25.1</version>
```

Comment:
I see, we can't use jersey.version here. Might be worth a note about why this dependency is here so that one day we might check and remove it when it's no longer needed. But as test scope it won't matter much.